src/sentry/options/defaults.py

```diff
@@ -2529,6 +2529,14 @@
     "api.organization-activity.brownout-duration", default="PT1M", flags=FLAG_AUTOMATOR_MODIFIABLE
 )
 
+
+register(
+    "sentry.save-event-attachments.project-per-5-minute-limit",
+    type=Int,
+    default=20000,
```

Comment:
i think we should make the default much much higher, or default infinity, and then ratchet it down from there

Reply:
Done